system/web/services/InterceptorService.cfc

```diff
@@ -259,6 +259,16 @@ component extends="coldbox.system.web.services.BaseService" accessors="true" {
 		return buffer;
 	}
 
+	/**
+	 * Unregister the given closure from a specific interception point
+	 *
+	 * @target The closure/lambda to unregister
+	 * @point  The interception point from which to unregister the listener
+	 */
+	void function unlisten( required target, required point ){
```

Comment:
Also, the `unregister` signature is a `boolean` return. So keep consistent and change it to `boolean` as well and just do a `return unregister()`